.storybook/theme/ThemeProvider.js

```diff
@@ -39,6 +59,15 @@ const theme = {
     l: '1rem',
     xl: '1.5rem',
   },
+  shadows: {
+    ...chakraDefaultTheme.shadows,
+    Card: {
+      isActive: `inset 0 0 0 3px ${colors.primary[500]}`,
+      0: `0 0 0 1px ${colors.element.inactive}`,
+      1: '0 2px 10px rgba(25, 1, 52, 0.08)',
+      2: '0 4px 20px rgba(25, 1, 52, 0.14)',
+    },
```

Comment:
I moved this here, thinking that it would be nice to have this kind of stuff centralized in the theme
Open to discussions 🙂